From be94aaec72dbacd0d948946ebab482864454b8ff Mon Sep 17 00:00:00 2001
Message-Id: <be94aaec72dbacd0d948946ebab482864454b8ff.1349639034.git.crobinso@redhat.com>
From: Kenneth Salerno <kennethsalerno@yahoo.com>
Date: Sat, 18 Feb 2012 16:05:44 -0800
Subject: [PATCH 1/3] qemu-1.0.1/VERSION

Hello,

The VERSION file in stable release qemu-1.0.1 has what I believe might be a typo: "1.0,1" rather than "1.0.1". This is causing a parsing issue for windres.exe in Win32 which chokes on:
   #define CONFIG_FILEVERSION 1,0,1,0,1,0
   #define CONFIG_PRODUCTVERSION 1,0,1,0,1,0

when it should be seeing this:
   #define CONFIG_FILEVERSION 1,0,1,0
   #define CONFIG_PRODUCTVERSION 1,0,1,0

Patch:

Signed-off-by: Justin M. Forbes <jforbes@redhat.com>
Signed-off-by: Cole Robinson <crobinso@redhat.com>
---
 VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/VERSION b/VERSION
index b70c292..7dea76e 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-1.0,1
+1.0.1
-- 
1.7.11.4

